### PR TITLE
(jenkins) Update changelog, bugtracker, mailing list and download URL

### PR DESCRIPTION
**Pull request by NotMyFault**
## Description
The change proposed updates the bug tracker URL, adds a mailing list URL, changes the changelog URL from weekly to the LTS changelog and uses a non-deprecated download URL.

## Motivation and Context
Current changes are either wrong or incomplete, this PR aims to fix them.

## How Has this Been Tested?
Tested locally.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by pauby** [comment]
@JPRuskin think this is one for you? Shpuld we add you to the CODEOWNERS?

**Review by Windos — APPROVED** [review]
These changes all make sense to me, and saves going through redirects from old addresses (or going to the correct place in general when it comes to the issues page). Thanks for the submission @NotMyFault!

I will wait for @JPRuskin to cast his eye over this before merging.

**Comment by JPRuskin** [comment]
LGTM! Thanks for noticing this and getting it sorted, @NotMyFault 

> @JPRuskin think this is one for you? Should we add you to the CODEOWNERS?

I don't know of any reason this would need to be just me, rather than the standard community-maintainer-shuffle? Particularly if the changes are just domain/metadata related.
It's not a complex package (bar migration-logic), and I didn't write the original - I just happened to PR it in from the previous owner.

**Review by JPRuskin — APPROVED** [review]

**Pull request merged**
